### PR TITLE
Added personal-values route

### PR DESCRIPTION
**Pull request by seanmajorpayne**
You can access this route at:

`/personal_values?session-id=1`

Returns top three values and their short descriptions. Since I couldn't find a full list of short descriptions anywhere, I've created temporary ones.

### CODE REVIEW

**Comment by biotom** [comment]
Looks good! Do you have any tests of the view, or API level tests for this endpoint?

File: knowledge_graph/routes.py [hunk 1]
```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
...
+
+@app.route('/personal_values', methods=['GET'])
+def get_personal_values():
+    """ Given a session-id, this returns the top three personal values for a user
+    
+    """
+    session_id = int(request.args.get('session-id'))
+    
+    try:
+        scores = db.session.query(Scores).filter_by(session_id=1).first()
```
Comment (biotom) [inline knowledge_graph/routes.py:181]: Why are we filtering by session_id = 1 here? shouldn't it be by the session id provided?

File: knowledge_graph/routes.py [hunk 2]
```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
 
     response = Response(dumps(value_scores))
     return response, 200
+    
+
+@app.route('/personal_values', methods=['GET'])
```
Comment (biotom) [inline knowledge_graph/routes.py:173]: Is it more readable to have the session-id parameter as a query string here?

File: knowledge_graph/routes.py [hunk 3]
```diff
@@ -152,6 +168,29 @@ def receive_user_scores() -> Tuple[Response, int]:
...
+def get_personal_values():
+    """ Given a session-id, this returns the top three personal values for a user
+    
+    """
+    session_id = int(request.args.get('session-id'))
+    
+    try:
+        scores = db.session.query(Scores).filter_by(session_id=1).first()
+    except:
```
Comment (biotom) [inline knowledge_graph/routes.py:182]: Is it worth specifying some exception types here? For instance- we could return invalid session ID for one exception, but internal server error for others? That might make diagnostics a bit easier in the future

File: knowledge_graph/routes.py (hunk 1 above)
Comment (seanmajorpayne) [reply knowledge_graph/routes.py:181]: Whoops, I forgot to change that back. I'll fix.

File: knowledge_graph/routes.py (hunk 2 above)
Comment (seanmajorpayne) [reply knowledge_graph/routes.py:173]: What do you mean by this? Can you type some example code here?

File: knowledge_graph/routes.py (hunk 3 above)
Comment (seanmajorpayne) [reply knowledge_graph/routes.py:182]: Sure, I'll add some exceptions

File: knowledge_graph/routes.py (hunk 2 above)
Comment (biotom) [reply knowledge_graph/routes.py:173]: Sorry, ignore me! I had the syntax mixed up with something else, I thought you could put some parameter in the route name and have flask read it like 'personal_values?session-id=<parameter>', but this is right

File: knowledge_graph/routes.py (hunk 3 above)
Comment (biotom) [reply knowledge_graph/routes.py:180]: I'm not seeing these updated exceptions here- there's just a bare 'except'. Am I just not seeing them for some reason?

**Review by rodriguesk — APPROVED** [review]
Should probably have the description of the personal values in a separate JSON file so it can change as needed or get pulled from a database or content management system at a later date. 

Overall This is good Though, assuming it works with docker, so I approve.

**Comment by NickCallaghan** [comment]
> You can access this route at:
> 
> `/personal_values?session-id=1`
> 

@seanmajorpayne How would I test this in postman. Is there a working database and a know good session_id to see the response? I've tried session-id=1 and also a valid uuid and got the same response. 

 If session_id is a uuid should:  `/personal_values?session-id=1` not be an invalid call? 

Should there be a different response for record not found as opposed to invalid request format?

**Comment by seanmajorpayne** [comment]
> Should probably have the description of the personal values in a separate JSON file so it can change as needed or get pulled from a database or content management system at a later date.
> 
> Overall This is good Though, assuming it works with docker, so I approve.

Yes, I can pull that out of the main file. Just a temporary quick solution to get front-end moving.

**Pull request closed without merging**